--- /root/axiom/configure	2006-08-29 13:01:36.000000000 -0400
+++ /root/silver/trunk/axiom/configure	2006-08-05 08:41:49.000000000 -0400
@@ -14,30 +14,24 @@
   SYSNAME=`cat /etc/redhat-release` 
   if [ "$SYSNAME" = "Fedora Core release 3 (Heidelberg)" ] ; 
    then SYSNAME=fedora3
-  else if [ "$SYSNAME" = "Fedora Core release 5 (Bordeaux)" ] ; 
-   then SYSNAME=fedora5
-   fi
   fi
   echo SYSNAME=$SYSNAME
 fi
-if [ ! "$SYSNAME" = "fedora3" ] ;
- then 
-  if [ ! "$SYSNAME" = "fedora5" ] ;
-   then
-    SYSNAME=`uname -s`
-    echo $SYSNAME 
-    if [ "$SYSNAME" = "Linux" ] ; then SYSNAME=linux
-    elif  [ "$SYSNAME" = "MINGW32_NT-5.1" ] ; then SYSNAME=windows
-    elif  [ "$SYSNAME" = "SunOS" ] ; then SYSNAME=solaris9
-    elif  [ "$SYSNAME" = "freebsd" ] ; then SYSNAME=freebsd
-    else
-      echo Your system name is $SYSNAME
-      echo We do not know how to build for this kind of system
-      echo Send a note to axiom-developer@nongnu.org about it
-      echo
-      exit 0
-    fi
-  fi
+if [ ! "$SYSNAME" = "fedora3" ] ; 
+ then
+   SYSNAME=`uname -s`
+   echo $SYSNAME 
+   if [ "$SYSNAME" = "Linux" ] ; then SYSNAME=linux
+   elif  [ "$SYSNAME" = "MINGW32_NT-5.1" ] ; then SYSNAME=windows
+   elif  [ "$SYSNAME" = "SunOS" ] ; then SYSNAME=solaris9
+   elif  [ "$SYSNAME" = "freebsd" ] ; then SYSNAME=freebsd
+   else
+     echo Your system name is $SYSNAME
+     echo We do not know how to build for this kind of system
+     echo Send a note to axiom-developer@nongnu.org about it
+     echo
+     exit 0
+   fi
 fi
 
 }
