--- /root/axiom/src/graph/view2D/Makefile.pamphlet	2006-08-29 13:01:37.000000000 -0400
+++ /root/silver/trunk/axiom/src/graph/view2D/Makefile.pamphlet	2006-08-05 08:40:53.000000000 -0400
@@ -24,8 +24,6 @@
 # global include files shared by everyone
 GINC=	${SRC}/include
 
-LDFLAGS = ${LDF} ${STATIC} -lX11 -lm ${LDF}
-
 @
 \section{view2D code}
 \subsection{buttons2d}
@@ -281,13 +279,11 @@
 
 @
 \subsection{view2D}
-Renaud Rioboo pointed out that we should dynamically link Xpm
-rather than use static libraries since statics fail on Fedora 5.
 <<view2D>>=
 ${OUT}/view2D: ${VIEW2D_OBJS} ${GDRAW_OBJS} ${LIBFILES}
 	@ echo 34 linking ${OUT}/view2D
 	@ ${CC} ${VIEW2D_OBJS} ${GDRAW_OBJS} ${LIBFILES} \
-	 -o ${OUT}/view2D $(LDFLAGS) 
+	 -o ${OUT}/view2D ${XLIB}/libXpm.a $(LDFLAGS) 
 
 @
 \section{axiom.sty}
@@ -305,6 +301,7 @@
 <<environment>>
 
 CFLAGS  = ${CCF} -I${LINC} -I${GINC} -I${IN}
+LDFLAGS = ${LDF} ${STATIC} -lX11 -lm ${LDF}
 
 DOCFILES= ${DOC}/buttons2d.c.dvi    ${DOC}/control2d.c.dvi \
           ${DOC}/graph2d.c.dvi      ${DOC}/main2d.c.dvi \
