--- /root/axiom/src/interp/sockio.lisp.pamphlet	2006-08-29 13:01:39.000000000 -0400
+++ /root/silver/trunk/axiom/src/interp/sockio.lisp.pamphlet	2006-08-05 08:40:53.000000000 -0400
@@ -69,12 +69,6 @@
   (system:define-foreign-function :c 'NANQ :single)
 )
 
-@
-This was changed as of GCL-2.6.8pre2. It used to read:
-\begin{verbatim}
-  (defentry sock_get_float (int) (float "sock_get_float"))
-\end{verbatim}
-<<*>>=
 #+KCL
 (progn
   (clines "extern double plus_infinity(), minus_infinity(), NANQ();")
@@ -84,7 +78,7 @@
   (defentry sock_send_int (int int) (int "sock_send_int"))
   (defentry sock_get_string_buf (int string int) (int "sock_get_string_buf"))
   (defentry sock_send_string_len (int string int) (int "sock_send_string_len"))
-  (defentry sock_get_float (int) (double "sock_get_float"))
+  (defentry sock_get_float (int) (float "sock_get_float"))
   (defentry sock_send_float (int float) (int "sock_send_float"))
   (defentry sock_send_wakeup (int int) (int "sock_send_wakeup"))
   (defentry server_switch () (int "server_switch"))
