--- /root/axiom/src/algebra/perm.spad.pamphlet	2006-08-29 13:01:37.000000000 -0400
+++ axiom/src/algebra/perm.spad.pamphlet	2006-08-05 08:40:33.000000000 -0400
@@ -58,7 +58,7 @@
 \section{domain PERM Permutation}
 <<domain PERM Permutation>>=
 )abbrev domain PERM Permutation
-++ Authors: Johannes Grabmeier, Holger Gollan, Martin Rubey
+++ Authors: Johannes Grabmeier, Holger Gollan
 ++ Date Created: 19 May 1989
 ++ Date Last Updated: 2 June 2006
 ++ Basic Operations: _*, degree, movedPoints, cyclePartition, order,
@@ -97,7 +97,7 @@
       ++ the permutation p as a list of preimages and images, i.e
       ++ p maps {\em (rep.preimage).k} to {\em (rep.image).k} for all
       ++ indices k. Elements of \spad{S} not in {\em (rep.preimage).k}
-      ++ are fixed points, and these are the only fixed points of the
+      ++ are fixed points, and these are the only fixed points of the 
       ++ permutation.
     coercePreimagesImages : List List S   ->  %
       ++ coercePreimagesImages(lls) coerces the representation {\em lls}
@@ -172,7 +172,7 @@
 and images of moved points. I.e., fixed points do not occur in either of these
 lists. This enables us to compute the set of fixed points and the set of moved
 points easily.
- 
+
 Note that this was not respected in versions before [[patch--50]] of this
 domain.
 
@@ -299,14 +299,25 @@
 Note that before [[patch--50]] this read
 \begin{verbatim}
       coercePreimagesImages preImageAndImage ==
-       p : % := [preImageAndImage.1,preImageAndImage.2]
+      p : % := [preImageAndImage.1,preImageAndImage.2]
 \end{verbatim}
 causing bugs when computing [[movedPoints]], [[fixedPoints]], [[even?]],
 [[odd?]], etc., as reported in Issue~\#295.
 
 The other coercion facilities check for fixed points. It also seems that [[*]]
 removes fixed points from its result.
- 
+
+<<TEST PERM>>=
+  p := coercePreimagesImages([[1,2,3],[1,2,3]])
+  movedPoints p    -- should return {}
+  even? p          -- should return true
+  p := coercePreimagesImages([[0,1,2,3],[3,0,2,1]])$PERM ZMOD 4
+  fixedPoints p    -- should return {2}
+  q := coercePreimagesImages([[0,1,2,3],[1,0]])$PERM ZMOD 4
+  fixedPoints(p*q) -- should return {2,0}
+  even?(p*q)       -- should return false
+@
+
 <<domain PERM Permutation>>=
 
     movedPoints p == construct p.1
@@ -458,7 +469,7 @@
 @
 
 Up to [[patch--50]] we did not check for duplicates.
- 
+
 <<domain PERM Permutation>>=
     if S has Finite then
 
@@ -468,7 +479,7 @@
 @
 
 Up to [[patch--50]] we did not check for duplicates.
- 
+
 <<domain PERM Permutation>>=
       fixedPoints ( p ) == complement movedPoints p
 
