--- /root/axiom/src/interp/Makefile.pamphlet	2006-08-29 13:01:38.000000000 -0400
+++ /root/silver/trunk/axiom/src/interp/Makefile.pamphlet	2006-08-05 08:40:52.000000000 -0400
@@ -3968,10 +3968,8 @@
 	  ${TANGLE} -RInterpreter ${IN}/bookvol5.pamphlet >bookvol5.${LISP} )
 
 @
-The new bookvol5 now depends on some pictures in the src/doc/ps
-subdirectory so we have to extend the preconditions to include this.
 <<bookvol5.dvi (DOC from IN)>>=
-${DOC}/bookvol5.dvi: ${IN}/bookvol5.pamphlet ${DOC}/axiom.sty ${DOC}/ps
+${DOC}/bookvol5.dvi: ${IN}/bookvol5.pamphlet ${DOC}/axiom.sty
 	@echo 299 making ${DOC}/bookvol5.dvi from ${IN}/bookvol5.pamphlet
 	@(cd ${DOC} ; \
 	cp ${IN}/bookvol5.pamphlet ${DOC} ; \
@@ -3987,12 +3985,6 @@
 	@ cp ${DOC}/bookvol5.dvi ${BOOK}/bookvol5.dvi
 
 @
-<<ps>>=
-${DOC}/ps: ${SRC}/doc/ps
-	@echo 299a making ${DOC}/ps from ${SRC}/doc/ps
-	@cp -pr ${SRC}/doc/ps ${DOC}
-
-@
 
 \subsection{i-intern.boot}
 <<i-intern.o (OUT from MID)>>=
@@ -8038,8 +8030,6 @@
 <<bookvol5.dvi (DOC from IN)>>
 <<bookvol5.dvi (BOOK from DOC)>>
 
-<<ps>>
-
 <<i-intern.o (OUT from MID)>>
 <<i-intern.clisp (MID from IN)>>
 <<i-intern.boot.dvi (DOC from IN)>>
