--- /root/axiom/src/interp/hash.lisp.pamphlet	2006-08-29 13:01:39.000000000 -0400
+++ /root/silver/trunk/axiom/src/interp/hash.lisp.pamphlet	2006-08-05 08:40:52.000000000 -0400
@@ -80,17 +80,8 @@
 #+Lucid
 (define-function 'HASHTABLE-CLASS #'system::hash-table-test)
 
-@
-The static declaration causes a problem as of GCL-2.6.8pre.
-Camm issued a fix. This used to read:
-\begin{verbatim}
 #+AKCL
 (clines "static int mem_value(x ,i)object x;int i; { return ((short *)x)[i];}")
-\end{verbatim}
-<<*>>=
-#+AKCL
-(clines "int mem_value(x ,i)object x;int i; { return ((short *)x)[i];}")
-
 #+AKCL
 (defentry memory-value-short(object int) (int "mem_value"))
 
