--- /root/axiom/src/doc/book.pamphlet	2006-08-29 13:01:37.000000000 -0400
+++ /root/silver/trunk/axiom/src/doc/book.pamphlet	2006-08-05 08:41:48.000000000 -0400
@@ -1234,7 +1234,7 @@
 information but Axiom will not do so automatically.
 
 \subsection{Type Conversion}
-To obtain the floating point value of a fraction one must convert 
+To obtain the floating point value of a fraction one must convert
 ({\bf conversions} are applied by the user and 
 {\bf coercions} are applied automatically by the interpreter) the result
 to type {\tt Float} using the ``::'' operator as follows: 
@@ -3316,7 +3316,7 @@
 \end{verbatim}
 \begin{verbatim}
 x:=1
-\end{verbatim}
+\begin{verbatim}
 $$
 1
 $$
@@ -3439,6 +3439,11 @@
 for i in 1..10 repeat
   ~prime?(i) => iterate
   output(i)
+\end{verbatim}
+\begin{verbatim}
+for i in 1..10 repeat
+  ~prime?(i) => iterate
+  output(i)
  
    2
    3
@@ -3450,6 +3455,10 @@
 \begin{verbatim}
 for w in ["This", "is", "your", "life!"] repeat
   output(w)
+\end{verbatim}
+\begin{verbatim}
+for w in ["This", "is", "your", "life!"] repeat
+  output(w)
  
    This
    is
@@ -3468,6 +3477,10 @@
 \begin{verbatim}
 for i in 1..10 | prime?(i) repeat
   output(i)
+\end{verbatim}
+\begin{verbatim}
+for i in 1..10 | prime?(i) repeat
+  output(i)
  
    2
    3
@@ -3479,6 +3492,10 @@
 \begin{verbatim}
 for i in [1,2,3,4,5,6,7,8,9,10] | prime?(i) repeat
   output(i)
+\end{verbatim}
+\begin{verbatim}
+for i in [1,2,3,4,5,6,7,8,9,10] | prime?(i) repeat
+  output(i)
  
    2
    3
@@ -3491,6 +3508,10 @@
 \begin{verbatim}
 for i in 1.. while i < 7 repeat
   if even?(i) then output(i)
+\end{verbatim}
+\begin{verbatim}
+for i in 1.. while i < 7 repeat
+  if even?(i) then output(i)
  
    2
    4
@@ -3502,6 +3523,10 @@
 \begin{verbatim}
 for i in 1.. | even?(i) while i < 7 repeat
   output(i)
+\{verbatim}
+\begin{verbatim}
+for i in 1.. | even?(i) while i < 7 repeat
+  output(i)
  
    2
    4
@@ -3514,6 +3539,10 @@
 \begin{verbatim}
 for a in 1..4 for b in 5..8 repeat
   output [a,b]
+\end{verbatim}
+\begin{verbatim}
+for a in 1..4 for b in 5..8 repeat
+  output [a,b]
  
    [1,5]
    [2,6]
@@ -3531,6 +3560,11 @@
 for a in 1..4 repeat
   for b in 7..9 | prime?(a+b) repeat
     output [a,b,a+b]
+\end{verbatim}
+\begin{verbatim}
+for a in 1..4 repeat
+  for b in 7..9 | prime?(a+b) repeat
+    output [a,b,a+b]
  
    [2,9,11]
    [3,8,11]
@@ -3545,6 +3579,10 @@
 \begin{verbatim}
 for a in 1..4 for b in 8..5 by -1 repeat
   output [a,b]
+\end{verbatim}
+\begin{verbatim}
+for a in 1..4 for b in 8..5 by -1 repeat
+  output [a,b]
  
    [1,8]
    [2,7]
